src/endpoints/stable-diffusion.js

```diff
@@ -550,7 +557,13 @@ comfy.post('/generate', jsonParser, async (request, response) => {
             await delay(100);
         }
         if (item.status.status_str === 'error') {
-            throw new Error('ComfyUI generation did not succeed.');
+            // Report node tracebacks if available
+            const errorMessages = item.status?.messages
```

Comment:
I have one for .messages on following L562... and a bonus one on status? which is probably too conservative since item.status.status_str has been fine for months